Cingulara/openrmf-api-controls
# release

- updating the version
- updated README
- #23 updating the Makefile version
- #23 adding cache directives to the API calls
- #22 updating the version for the changes
- #22 Fixing the name of the API in Grafana
- Add tests for GH Actions
- adding Jaeger Tracing framework
- updating to latest POAM changes
- #18 giving an index sort order
- #18 major controls API endpoint
- putting the latest version tag in here
- Adding Jaeger Tracing Information
- removing old code not needed
- updated copyright
- adding Prometheus Metrics export
- updating the copyright notice
- updated logging for API calls
- updating the API documentation
- updating the copyright notice
- Adding NATS options to connect
- Update Makefile to version 0.10
- Update issue templates
- fixing naming in the README
- #8 fixing bad namespace
- clean up tests
- fixing server ENV to all caps
- updating for using NATS for api-to-api comms
- fixing code for RBAC and API to API comms
- updates for RBAC AuthN and AuthZ with keycloak
- updating for tests
- adding "always include" controls
- finalizing the PII listing
- adding in pii filter
- Updating tests to run on controllers
- updated model tests
- Getting test classes started
- reorganized for Moq unit tests
- Updating listing for PII
- Compliance for low/moderate/high
- updated changes to openRMF
- making dockerhub quicker
- updating to match version 0.6
- adding remaining missing items for NIST controls
- updating missing controls on PII and privacy
- fixing XML for parsing extra families
- adding extra controls for PII
- removing unneeded code
- adjusting search individual for family if 404
- Fix ending numbers with a period for matching
- updating the loading of enhancements
- fixing the formatting of sub controls
- fixing formatting of data
- generate PK ID and save often when loading
- Fixing the loading of initial controls at startup
- making a flat table for the database in memory
- controls listing in memory db -- need to flatten
- initially getting the controls loaded into memory
